### PR TITLE
[ES] Community rules update

### PR DESCRIPTION
**Pull request by Chrysheight**
### Description

Spanish update for the new community rules.

### Status

Pending review.

### Changelog

Empty

### Related Issues 

~~All the links points to english related osu pages, this has to be fixed once the respective pages have been translated to Spanish, for the sake of consistency.~~
There are some conflicts on the original English Community Rules page that go against the Article Styling Criteria mentioned above that might require some attention.

### CODE REVIEW

File: wiki/Rules/es.md [hunk 1]
```diff
@@ -1,76 +1,73 @@
...
+  8.  **No promociones servidores de chat de terceros de ningún tipo, incluyendo servidores/grupos de Discord, Teamspeak o Skype.** El chat de osu! es principalmente sobre *osu!* y no sobre compartir links a tus servidores personales. Hazlo a través de mensajes privados (MP) si es que tienes que hacerlo.
+  9.  **En los canales de idiomas específicos, debes hablar el idioma dominante (inglés en `#english`, tailandés en `#thai`, etc) si es que estás participando en alguna discusión.** En el caso que el lenguaje de un canal no sea explícito (o inferido por el nombre del canal, como en tailandés para `#thai` u holandés para `#dutch`), este siempre será inglés. Los canales #lobby y #help en cualquier lenguaje, a diferencia de los otros canales. *Miembros del [personal de moderación](https://osu.ppy.sh/help/wiki/People/Global_Moderation_Team) no necesitan seguir esta regla, cuando hacen su trabajo.*
```
Comment (MegaApplePi) [inline wiki/Rules/es.md]: change this link `[personal de moderación](https://osu.ppy.sh/help/wiki/People/Global_Moderation_Team)` to
`[personal de moderación](/wiki/People/Global_Moderation_Team)`

File: wiki/Rules/es.md [hunk 2]
```diff
@@ -1,76 +1,73 @@
...
-2.  **Manténgase en el tema principal (mods y ayudas) en la discusión del Beatmap.** Estos temas/discusiones son sobre el Beatmap en cuestión y nada mas. Si tienes un problema con algo que no es del Beatmap, publícalo en el foro/subforo apropiado.
-3.  **No cometas plagio o robes el trabajo de otros.** No incluyas ni uses el trabajo de otros usuarios sin el adecuado permiso (incluyendo pero no limitado a Skins y GDs).
+  Estas reglas aplican estríctamente a los beatmaps que sean presentados a través del [Beatmap Submission System](https://osu.ppy.sh/help/wiki/Glossary/#bss) (Sistema de Envio de Beatmaps) dentro del juego. Estos hilos/discusiones son creados a través del cliente osu! cuando un beatmap es presentado a través del [editor](https://osu.ppy.sh/help/wiki/Beatmap_Editor).
```
Comment (MegaApplePi) [inline wiki/Rules/es.md]: change this link `[Beatmap Submission System](https://osu.ppy.sh/help/wiki/Glossary/#bss)` to
`[Beatmap Submission System](/wiki/Glossary/#bss)`

Comment (MegaApplePi) [inline wiki/Rules/es.md]: change this link `[editor](https://osu.ppy.sh/help/wiki/Beatmap_Editor)` to
`[editor](/wiki/Beatmap_Editor)`

File: wiki/Rules/es.md [hunk 3]
```diff
@@ -1,76 +1,73 @@
...
 
-Reglas de Off-Topic
----------------
+  Existe un set de reglas más detallado para la construcción de beatmaps, conocido como Los [Ranking Criteria](https://osu.ppy.sh/help/wiki/Ranking_Criteria) (Criterios de Ranking).
```
Comment (MegaApplePi) [inline wiki/Rules/es.md]: change this link `[Ranking Criteria](https://osu.ppy.sh/help/wiki/Ranking_Criteria)` to
`[Ranking Criteria](/wiki/Ranking_Criteria)`

**Comment by MegaApplePi** [comment]
All links are technically pointing to the English articles, but this is intentional because when you view the article live on the site, it will give you the correct language based on your selected language.

File: wiki/Rules/es.md [hunk 4]
```diff
@@ -1,76 +1,73 @@
...
-3.  **No seas un idiota.** Si en algún momento no estás seguro de si estás infringiendo las reglas, probablemente lo estés haciendo.
-4.  **Somos una comunidad para todas las edades.** Esto significa que todo contenido para mayores de 18 años y/o NSFW (contenido explicito) como drogas o temas sexuales no son bienvenidos aquí, excepto cuando se hace un comentario apropiado para eso, pero generalmente hablando, manténganlo limpio.
-5.  **Donde las reglas no prevalecen, Usa el sentido común.** La administración tiene la explicita discreción de aplicar su juicio así sea necesario.
+  1.  **Cada jugador puede tener sólamente UNA cuenta, en todo momento.** La *primera* cuenta que creas durante el regstro de cuenta será tu única cuenta de *osu!*, desde el momento de su creación para siempre jamás. Ésta cuenta TUYA. No es de nadie más, (no es de tu hermano, tu madre, tu hermana ni de tu amigo) es TUYA. No compartas tu cuenta con nadie más. No podrás hacer una nueva si pierdes acceso a tu cuenta. Mantenla segura.
```
Comment (ImMock) [inline wiki/Rules/es.md]: Está mal escrito "regstro" y toda la linea puede quedar un poco mejor si quitas:

...durante el registro **de cuenta** será tu única cuenta...

Quedaría;  La *primera* cuenta que creas durante el registro será tu única cuenta de *osu!*

Comment (Chrysheight) [reply wiki/Rules/es.md]: Arreglado, gracias!

**Comment by TPGPL** [comment]
Change the title

**Comment by MegaApplePi** [comment]
@Chrysheight Are you all done with this? If so, can you squash your commits and use a better commit name?

**Pull request closed without merging**